Idea.js

```diff
@@ -12,10 +12,12 @@ class Idea {
     localStorage.setItem(stringifiedId, stringifiedNewIdea);
   }
 
-  deleteFromStorage(idea) {
-    idea.localStorage.removeItem(idea.id);
+  deleteFromStorage() {
```

Comment:
I think this now has a for loop in the main. We'll see if any merge conflicts arise and choose the newest one.